modules/storage/src/storage-provider/interfaces/IStorageProvider.ts

```diff
@@ -1,8 +1,6 @@
 export interface IStorageProvider {
   store(fileName: string, data: any, isPublic?: boolean): Promise<boolean | Error>;
 
-  get(fileName: string, downloadPath?: string): Promise<any | Error>;
```

Comment:
Because it is defined repeatedly with line 48

https://github.com/ConduitPlatform/Conduit/blob/289e0f904d2115d4f055855e75f0b26e469ea0df/modules/storage/src/storage-provider/interfaces/IStorageProvider.ts#L35

Reply:
Oops, you're right about that 👍